vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -279,6 +280,7 @@
   public Map() {
     getView();
     theMap.addMouseListener(this);
+    theMap.addMouseMotionListener(this);
```

Comment:
What's the deal with this?

Reply:
I dunno -- it seemed like it *should* be there. This particular bit is one of a couple of things where I *thought* I was going go need some support in Map.java but then it turned out I didn't, but I left some of the cleanupy things in.